src/components/BooksPage.js

```diff
@@ -1,31 +1,37 @@
-import React, { useState } from 'react';
-import BookList from './BookList';
-import BookForm from './BookForm';
-import Navigation from './Navigation';
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { v4 as uuidv4 } from 'uuid'; // Import uuidv4 function
+import BookItem from './BookItem'; // Import your BookItem component
+import { addBook, removeBook } from '../redux/books/booksSlice'; // Import your actions
 
 function BooksPage() {
-  const [books, setBooks] = useState([]);
+  const books = useSelector((state) => state.books);
+  const dispatch = useDispatch();
 
-  const handleAddBook = (newBook) => {
-    setBooks([...books, newBook]);
+  const handleAddBook = () => {
+    const newBook = {
+      item_id: uuidv4(), // Use uuidv4 to generate a unique ID
+      title: 'New Book Title',
+      author: 'New Book Author',
+      category: 'Fiction',
+    };
+    dispatch(addBook(newBook));
   };
 
-  const handleDeleteBook = (bookId) => {
-    const updatedBooks = books.filter((book) => book.id !== bookId);
-    setBooks(updatedBooks);
+  const handleRemoveBook = (bookId) => {
+    dispatch(removeBook(bookId));
   };
 
   return (
     <div>
-      <Navigation />
-      {' '}
-      {/* Display navigation links */}
-      <BookList books={books} onDeleteBook={handleDeleteBook} />
-      {' '}
-      {/* Display book list */}
-      <BookForm onAddBook={handleAddBook} />
-      {' '}
-      {/* Display book form */}
+      {books.map((book) => (
+        <div key={book.item_id}>
+          <BookItem book={book} />
+          <button onClick={() => handleRemoveBook(book.item_id)}>Remove Book</button>
+        </div>
+      ))}
+
+      <button onClick={handleAddBook}>Add Book</button>
```

Comment:
- [ ] according to the requirements there should be a separate component for adding the books, this component should have a local state for title and author, which is updated through an onChange event and the button should add updated local state to the redux store.